docs/source/creating_an_image_processing_pipeline/index.rst

```diff
@@ -75,20 +75,6 @@ origin. At this point, it's trivial to create a cell x gene matrix.
 
     tutorials/exec_feature_identification_and_annotation.rst
 
-Putting Together a Pipeline Recipe and Running it
```

Comment:
Are we dropping the recipe runner?

Reply:
I took out all the recipe stuff :/ should leave the runner in?